scripts/lib/CIME/tests/test_sys_cmake_macros.py

```diff
@@ -28,6 +29,9 @@ def xml_to_tester(self, xml_string):
     def setUp(self):
         super().setUp()
 
+        if "CIME_NO_CMAKE_MACRO" not in os.environ:
```

Comment:
Don't test the deprecated system unless the user indicates they want the deprecated system in their env.